composer.json

```diff
@@ -12,7 +12,7 @@
     "pfrenssen/phpcs-pre-push": "1.0",
     "phing/phing": "~2",
     "wimg/php-compatibility": "~7.0.7",
-    "ext-tidy": "^2.0"
+    "ext-tidy": "^2.0|^5.6|^7"
```

Comment:
`"ext-tidy": "*"` is the way to do it for PHP extensions.

See: https://github.com/search?l=JSON&q=ext-tidy&type=Code

Reply:
Fixed